src/render/draw_data.rs

```diff
@@ -68,14 +69,14 @@ impl DrawData {
 
 /// Iterator over draw lists
 pub struct DrawListIterator<'a> {
```

Comment:
This type no longer needs to exist (it's simply a wrapper for slice::Iter currently), but I'm keeping it to avoid changing the interface. Could be replaced with
```rust
/// Iterator over draw lists
pub type DrawListIterator<'ui, 'a> = std::slice::Iter<'a, DrawList<'ui>>;
```